test/models/test_layout.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python
+"""
+<Program Name>
+  test_layout.py
+
+<Author>
+  Santiago Torres-Arias <santiago@nyu.edu>
+
+<Started>
+  Nov 18, 2016
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
+  Test layout class functions.
+
+"""
+
+import unittest
+import datetime
+from toto.models.layout import Layout, Step, Inspection
+from toto.ssl_commons.exceptions import FormatError
+import toto.ssl_crypto
+
+class TestLayoutValidator(unittest.TestCase):
+  """Test verifylib.verify_delete_rule(rule, artifact_queue) """
+
+  def setUp(self):
+    """ Populate a base layout that we can use """
+    self.layout = Layout()
+    self.layout.expires = datetime.datetime.now().isoformat()
```

Comment:
Please check out this [comment]( https://github.com/in-toto/in-toto/pull/48/files#diff-03000fa12a2e142e6c48c5bff130c268R156).